test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -327,6 +329,26 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    @BeforeClass
+    public static void ensureSupportedLocale() {
+        if (isUnusableLocale()) {
+            // See: https://github.com/bcgit/bc-java/issues/405
+            Logger logger = LogManager.getLogger(ESTestCase.class);
+            logger.warn("Attempting to run tests in an unusable locale in a FIPS JVM. Certificate expiration validation will fail, " +
```

Comment:
Maybe it would be good to provide a link to the upstream bug so it's easier to check if this is still relevant ?

Reply:
In the log message you mean ? Sure thing

Reply:
yes, move it to the log message rather than the comment.